src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -7,30 +7,35 @@ import React, {
   useState,
 } from 'react';
 import PropTypes from 'prop-types';
+import useFetch from '@bloodyaugust/use-fetch';
 import Select from '../select/select.jsx';
 import ListOption from '../list-option/list-option.jsx';
+import mockConstants from '../../mocks/mock-constants.js';
+
+const { API_ROOT } = mockConstants;
 
 const CustomFieldInputSingleChoice = forwardRef(function CustomFieldInputSingleChoice(props, ref) {
+  const [choices, setChoices] = useState([]);
+  const [listOptionRefs, setListOptionRefs] = useState([]);
```

Comment:
This seem unnecessary. Try this:

```suggestion
  const listOptionRefs = choices.map(() => createRef());
```